hollow/src/main/java/com/netflix/hollow/api/custom/HollowTypeAPI.java

```diff
@@ -29,20 +30,14 @@
  *  This can be useful in tight loops, where the excess object creation incurred by using a Generated or Generic
  *  Hollow Object API would be prohibitively expensive.
  */
-public abstract class HollowTypeAPI {
+public abstract class HollowTypeAPI implements HollowRecordDelegate {
 
-    protected final HollowAPI api;
     protected final HollowTypeDataAccess typeDataAccess;
 
-    protected HollowTypeAPI(HollowAPI api, HollowTypeDataAccess typeDataAccess) {
-        this.api = api;
+    protected HollowTypeAPI(HollowTypeDataAccess typeDataAccess) {
         this.typeDataAccess = typeDataAccess;
     }
 
-    public HollowAPI getAPI() {
```

Comment:
Removing this method is a breaking change. Did you remove to simplify or was it required for all type API implementations to implement their corresponding delegate?